### PR TITLE
Use one click tests instead of the manual setup and run

### PR DESCRIPTION
**Pull request by thomastaylor312**

### CODE REVIEW

**Comment by thomastaylor312** [comment]
This is ready to go. Looks like dropping it in was as simple as I had hoped thanks to @itowlson's great work on the testing stuff

**Review by bacongobbler — APPROVED** [review]
oh this looks wonderful. Love that it would be removing all of this setup.

**Comment by thomastaylor312** [comment]
I'm doing some debugging here still unfortunately. I always forget that GH actions doesn't use the context of this PR to run tests, so I am testing on my fork to make sure this actually works

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -1,101 +0,0 @@
-name: Build and Test
-on:
-  pull_request:
```
Comment (thomastaylor312) [inline .github/workflows/build.yml:3]: This was the exact same as the other workflow file, just with different events. I combined the events and deleted this file

Comment (thomastaylor312) [reply .github/workflows/build.yml]: And then I remembered that we did this for a reason

**Comment by thomastaylor312** [comment]
This now includes a fix for unintentionally dropped events that was causing issues with the e2e pipeline

**Review by bacongobbler — APPROVED** [review]

**Pull request merged**